tools/topology/topology2/production/tplg-targets-ace1.cmake

```diff
@@ -32,6 +32,12 @@ NUM_HDMIS=0,SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,
 SDW_DMIC_STREAM=Capture-SmartMic,SDW_JACK_OUT_STREAM=Playback-SimpleJack,\
 SDW_JACK_IN_STREAM=Capture-SimpleJack,SDW_AMP_FMT_24=true,SDW_JACK_FMT_24=true"
 
+"cavs-sdw\;sof-mtl-cs42l43-l0-cs35l56-l23\;PLATFORM=mtl,NUM_SDW_AMP_LINKS=2,SDW_DMIC=1,\
+NUM_HDMIS=0,SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,\
```

Comment:
Shouldn't `SDW_AMP_FEEDBACK` be true? cs35l56 supports feedback.

Reply:
As far as I know cs35l56's feedback is only used for debugging so shouldn't be used for "production" (@rfvirgil please correct if wrong)

Reply:
> As far as I know cs35l56's feedback is only used for debugging so shouldn't be used for "production" (@rfvirgil please correct if wrong)

If so, we will need to remove the feedback from sof-mtl-cs42l43-l0-cs35l56-l12, too. Let's wait for @rfvirgil's confirmation.